### ISSUE TITLE
Used haku33 to reset, now banned

### ISSUE THREAD

**Issue by weirdal3333** [StarDustCFW/Haku33#30]
I reset my switch with haku33, to start over. The system worked fine and was able to connect to and use Nintendo services for a few weeks, but is now now banned. 
I have been using an emunand with incognito and exosphere product blanking for years to prevent a ban from happening.
I have used cfw on the nand before, and didn't have a clean backup.
That is why I used haku33.

**Comment by Kronos2308** [comment]
Yes that's a real risk 
I sorry that my soft not help you 
But any trace left was removed but meaby the emunand connects in some moment to the internet
 Or the blanking fail for some reason

**Comment by Kronos2308** [comment]
I will not close this
There is always a risk using not official software
 And people should be aware off that

Now since is no way to unban a banned switch and the cleaning method has not change over the time, there's is not much I can do

I appreciate your feedback

**Comment by weirdal3333** [comment]
Yes, thank you! I have an idea on why the console was banned. The time was set to the year 5507 in the past somehow, and the console sent logs with that date after resetting the console. Haku33 did not reset the system time:
![20240226_073508](https://github.com/StarDustCFW/Haku33/assets/3537180/9fff8ac3-45d2-4b34-8245-704c3b9f04f7)
